sql/core/src/test/resources/sql-tests/results/ansi/try_element_at.sql.out

```diff
@@ -7,8 +7,8 @@ SELECT try_element_at(array(1, 2, 3), 0)
 -- !query schema
 struct<>
 -- !query output
-java.lang.ArrayIndexOutOfBoundsException
-SQL array indices start at 1
+org.apache.spark.SparkArrayIndexOutOfBoundsException
+[INVALID_ARRAY_INDEX_IN_ELEMENT_AT] The index 0 is out of bounds. The array has 3 elements. Use `try_element_at` to tolerate accessing element at invalid index and return NULL instead. If necessary set "spark.sql.ansi.enabled" to "false" to bypass this error.
```

Comment:
> Use `try_element_at` to tolerate

The query uses `try_element_at` already. The recommendation is not useful again.

Reply:
Yes, i changed it.